azure-pipelines.yml

```diff
@@ -8,8 +8,12 @@ jobs:
 - job: Linux
   strategy:
     matrix:
-      py35-min:
-        conda_env: py35-min
+      py35-bare-minimum:
+        conda_env: py35-bare-minimum
+      py36-min:
+        conda_env: py36-min
```

Comment:
Do we need a py36-min as well as a py35-min?

Reply:
maybe `py36-min-all-deps` would be a more descriptive alternative name.

Reply:
done